NorthstarDedicatedTest/dllmain.cpp

```diff
@@ -67,6 +77,96 @@ void WaitForDebugger(HMODULE baseAddress)
 			Sleep(100);
 	}
 }
+bool LoadPlugins() {
+
+	std::vector<fs::path> paths;
+
+	std::string pluginPath = GetNorthstarPrefix() + "/plugins";
```

Comment:
IIRC, @BobTheBob9 wanted this in bin.

Reply:
We talked about it and either one was fine.